aws/runSentryRelease.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+[[ -n "${GENERATION_DEBUG}" ]] && set ${GENERATION_DEBUG}
+trap '. ${GENERATION_DIR}/cleanupContext.sh; exit ${RESULT:-1}' EXIT SIGHUP SIGINT SIGTERM
+. "${GENERATION_DIR}/common.sh"
+
+# Get the generation context so we can run template generation
+. "${GENERATION_DIR}/setContext.sh"
+
+function usage() {
+    cat <<EOF
+
+Upload sourcemap files to sentry for a specific release
+
+Usage: $(basename $0) -s SENTRY_SOURCE_MAP_S3_URL -r SENTRY_RELEASE
+
+where
+
+    -h                              shows this text
+(m) -s SENTRY_SOURCE_MAP_S3_URL     s3 link to sourcemap files
+(m) -r SENTRY_RELEASE_NAME          sentry release name
+
+(m) mandatory, (o) optional, (d) deprecated
+
+DEFAULTS:
+
+NOTES:
+
+EOF
+    exit
+}
+
+function options() { 
+
+    # Parse options
+    while getopts ":hs:r:" opt; do
+        case $opt in
+            h)
+                usage
+                ;;
+            s)
+                SENTRY_SOURCE_MAP_S3_URL="${OPTARG}"
+                ;;
+            r)
+                SENTRY_RELEASE_NAME="${OPTARG}"
+                ;;
+            \?)
+                fatalOption
+                ;;
+            :)
+                fatalOptionArgument
+                ;;
+        esac
+    done
+
+}
+
+
+function main() {
+
+  options "$@" || return $?
+
+  yarn global add @sentry/cli || return $?
```

Comment:
Might be safer to make this optional so that we choose when to update the sentry-cli version or lock it to a major version